router/route.go

```diff
@@ -13,19 +11,16 @@ import (
 
 // Routes load middlewares
 func Routes(middleware ...gin.HandlerFunc) http.Handler {
-	var (
-		router    = gin.Default()
-		jwtSecret = os.Getenv("JWT_SECRET")
-	)
+	var router = gin.Default()
```

Comment:
I realize this may be a little late, but are you OK with using gorilla mux for routing instead?